homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -127,6 +127,13 @@
     }
 )
 
+SERVICE_SCHEMA_GOTO = VACUUM_SERVICE_SCHEMA.extend(
```

Comment:
We don't need to include the entity id in the schema. Just pass a dict with the voluptous validators for the parameters that are not entity, inline when registering the service. See the example in the docs.

Reply:
Okay. The schema maps are removed and schema is now passed inline as dict without enity_id attribute to register_service